core/src/window_service.rs

```diff
@@ -218,38 +217,6 @@ fn run_check_duplicate(
         .try_for_each(check_duplicate)
 }
 
-fn verify_repair(
-    outstanding_requests: &mut OutstandingShredRepairs,
-    shred: &Shred,
-    repair_meta: &Option<RepairMeta>,
-) -> bool {
-    repair_meta
-        .as_ref()
-        .map(|repair_meta| {
-            outstanding_requests
-                .register_response(
-                    repair_meta.nonce,
-                    shred,
-                    solana_sdk::timing::timestamp(),
-                    |_| (),
-                )
-                .is_some()
-        })
-        .unwrap_or(true)
```

Comment:
Just to confirm, the `.unwrap_or(true)` was here because `WindowService` is operating on turbine & repair shreds, and so `repair_meta.is_some() == false` implies this is a turbine shred ?

Reply:
right